app/templates/main_nav.html

```diff
@@ -18,16 +18,19 @@
         <li><a href="{{ url_for('main.uploads', service_id=current_service.id) }}" {{ casework_navigation.is_selected('uploads') }}>Uploads</a></li>
       {% endif %}
     {% endif %}
-  {% endif %}
     <li><a href="{{ url_for('.manage_users', service_id=current_service.id) }}" {{ main_navigation.is_selected('team-members') }}>Team members</a></li>
-  {% if current_user.has_permissions('manage_service', allow_org_user=True) %}
+    {% if current_user.has_permissions('manage_service', allow_org_user=True) %}
```

Comment:
No, it’s effectively unreachable code
```suggestion
    {% if current_user.has_permissions('manage_service') %}
```

Reply:
👍

Reply:
Actually, the above comment is wrong. We do need this check because:
- if you have permission to see the service
- but are also a user of the organisation
- but don’t have the `manage_service` permission
- you should always be able to see the usage page

Reply:
I'm a little confused. If they don't have the `manage_service` permission though then this whole statement will be false anyway so they won't see the Usage page?

I could see it adding users who have BOTH the manage_service permission and are also org users (if that's possible, I'm very unfamiliar with permissions stuff) but not one who doesn't have the`manage_service` permission?

I'm going to have a look now to see if removing the allow_org_user flag in this case breaks a test

Reply:
I’ve added an explicit test for this in b7b9e97. The test fails if the above suggestion is applied:

![image](https://user-images.githubusercontent.com/355079/73669437-55b6a900-469f-11ea-985a-f85d62364664.png)

Reply:
Oh cool, you were already thinking along the same lines. I'll take another review